common/include/pcl/point_types_conversion.h

```diff
@@ -134,6 +136,58 @@ namespace pcl
     if (out.h < 0.f) out.h += 360.f;
   }
 
+  /** \brief Convert a XYZRGBA point type to a XYZLAB
```

Comment:
I think taking XYZRGB as primary would be better. XYZRGBA has a field that has no corresponding field in CIE L*a*b*

Reply:
Great idea, thank you! I'd change this function for XYZRGB conversion. What about the function call of `exact XYZRGBA`->`XYZLAB` in gicp6d.cpp?
https://github.com/PointCloudLibrary/pcl/blob/9a912385c8b5a8109c91b19a36a1fe4f05e6e6c5/registration/src/gicp6d.cpp#L65
I have two plans but tentative about which one is better:
- wrap XYZRGB conversion for another XYZRGBA->XYZLAB
- or, make XYZRGB conversion a templated function for all possible XYZRGBA point types as input

Reply:
PCL basically acts like RGBA is RGB (ignores the A) everywhere else. So either we need a XYZRGBA + XYZRGB functions (with XYZRGBA being a wrapper), or just XYZRGB function that can accept XYZRGBA

Reply:
Thanks for your reviewing! I've made XYZRGB2XYZLAB a template function for all XYZRGB-based point types.